interview-questions.md

```diff
@@ -4,49 +4,49 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What does CRUD stand for?
 
-  Your answer:
+  Your answer: Create, read, update, destroy
 
-  Researched answer:
+  Researched answer: The four basic functions of persistent storage
 
 
 
 2. What are the 5 HTTP verbs?
 
-  Your answer:
+  Your answer: Patch, put, delete, get, update
 
-  Researched answer:
+  Researched answer: Post, Get, Put, Patch, and Delete
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
-  Your answer:
+  Your answer: We need to have import our react dependicies, class decleration, render method, and class export 
 
-  Researched answer:
+  Researched answer: We will need a div component to store the string, import function, render method, and class decleration
```

Comment:
this question was a little confusing and your answer is great. but just so you know, we were looking for: import, class name extends, render, return, div/React.Fragment would be a bonus, export included in this answer somehow